<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_SCOPPFoot</class>
 <widget class="QMainWindow" name="GUI_SCOPPFoot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SCOPPFoot</string>
  </property>
  <widget class="QWidget" name="qwdg_central">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="qfrm_central">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolBox" name="qttbx_acciones">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>158</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>158</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="qttpg_modelar">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>158</width>
              <height>222</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <attribute name="label">
             <string extracomment="Aquí se codificaran las tareas para el entrenamiento del modelo de clasificación de proteínas">Entrenar Modelo</string>
            </attribute>
            <attribute name="toolTip">
             <string>Tareas para el entrenamiento de modelos de clasificación de proteínas</string>
            </attribute>
            <layout class="QVBoxLayout" name="vlayBotonesPricipales">
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="qcbl_selecDataSet">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>38</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>154</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selecciona la ubicación del dataset para entrenar el modelo de clásificación&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Seleccionar Dataset</string>
               </property>
               <property name="icon">
                <iconset resource="resources/img/imagenes.qrc">
                 <normaloff>:/entrenarModelo/proteina.png</normaloff>:/entrenarModelo/proteina.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>true</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="qcbl_caracterizarDataset">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>38</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>154</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selecciona la ubicación del dataset para entrenar el modelo de clásificación&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>Procesar Dataset</string>
               </property>
               <property name="icon">
                <iconset resource="resources/img/imagenes.qrc">
                 <normaloff>:/entrenarModelo/kmeans.png</normaloff>:/entrenarModelo/kmeans.png</iconset>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="qttpg_clasificar">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>158</width>
              <height>222</height>
             </rect>
            </property>
            <attribute name="label">
             <string extracomment="Aquí se codificaran las tareas para la clasificación de una proteína o un cojunto de prueba de proteínas.">Clasificar Proteína</string>
            </attribute>
            <attribute name="toolTip">
             <string>Aquí se realiza la clasificación de proteínas, con base en el modelo por defecto o el modelo previamente especificado.</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="qfrm_forms">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="lineaSepEstado">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="qhbl_estado">
        <item>
         <widget class="QProgressBar" name="qpbr_avanceTarea">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>23</height>
           </size>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="qmbr_princpl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="qmen_archivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="qact_cargarModelo"/>
    <addaction name="qact_cargarFeatures"/>
    <addaction name="qact_salvarFeatures"/>
    <addaction name="qact_salvarModelo"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionPreferencias"/>
   </widget>
   <addaction name="qmen_archivo"/>
   <addaction name="menuEditar"/>
  </widget>
  <action name="qact_cargarModelo">
   <property name="icon">
    <iconset resource="resources/img/imagenes.qrc">
     <normaloff>:/menu/cargarModelo.png</normaloff>:/menu/cargarModelo.png</iconset>
   </property>
   <property name="text">
    <string>Cargar Modelo</string>
   </property>
   <property name="toolTip">
    <string>Carga un modelo previamente entrenado para el clasificador</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="qact_salvarFeatures">
   <property name="text">
    <string>Salvar Features</string>
   </property>
  </action>
  <action name="qact_salvarModelo">
   <property name="text">
    <string>Salvar Modelo</string>
   </property>
  </action>
  <action name="qact_cargarFeatures">
   <property name="text">
    <string>Cargar Features</string>
   </property>
   <property name="toolTip">
    <string>Carga un conjunto de features previamente encontrados por la aplicación</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPreferencias">
   <property name="text">
    <string>Preferencias</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/img/imagenes.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qact_cargarModelo</sender>
   <signal>triggered()</signal>
   <receiver>GUI_SCOPPFoot</receiver>
   <slot>cargarModelo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
